source/routing/defining-your-routes.md

```diff
@@ -45,335 +45,48 @@ the case of `/`, the name `index`).
-automatically generate one for you.)
-
-Ember.js automatically figures out the names of the routes and controllers based on
-the name you pass to `this.route`.
-
-<table>
-  <thead>
-  <tr>
-    <th>URL</th>
-    <th>Route Name</th>
-    <th>
-      Controller<br/>
-      <code>app/controllers/</code>
-    </th>
-    <th>
-      Route<br/>
-      <code>app/routes/</code>
-    </th>
-    <th>
-      Template<br/>
-      <code>app/templates/</code>
-    </th>
-  </tr>
-  </thead>
-  <tr>
-    <td><code>/</code></td>
-    <td><code>index</code></td>
-    <td>↳<code>index.js</code></td>
-    <td>↳<code>index.js</code></td>
-    <td>↳<code>index.hbs</code></td>
-  </tr>
-  <tr>
-    <td><code>/about</code></td>
-    <td><code>about</code></td>
-    <td>↳<code>about.js</code></td>
-    <td>↳<code>about.js</code></td>
-    <td>↳<code>about.hbs</code></td>
-  </tr>
-  <tr>
-    <td><code>/favs</code></td>
-    <td><code>favorites</code></td>
-    <td>↳<code>favorites.js</code></td>
-    <td>↳<code>favorites.js</code></td>
-    <td>↳<code>favorites.hbs</code></td>
-  </tr>
-</table>
-
-### Nested Routes
+In these cases, you can use **nested routes** to display one template inside
+of another.
 
 You can define nested routes by passing a callback to `this.route`:
 
 ```app/router.js
 Router.map(function() {
-  this.route('posts', { path: '/posts' }, function() {
+  this.route('posts', function() {
     this.route('new');
   });
 });
 ```
 
-This router creates these routes:
-
-<table>
-  <thead>
-  <tr>
-   <th>URL</th>
-   <th>Route Name</th>
-   <th>
-     Controller<br/>
-     <code>app/controllers/</code>
-   </th>
-   <th>
-     Route<br/>
-     <code>app/routes/</code>
-   </th>
-   <th>
-     Template<br/>
-     <code>app/templates/</code>
-   </th>
-  </tr>
-  </thead>
-  <tr>
-    <td><code>/</code></td>
-    <td><code>index</code></td>
-    <td>↳<code>index.js</code></td>
-    <td>↳<code>index.js</code></td>
-    <td>↳<code>index.hbs</code></td>
-  </tr>
-  <tr>
-    <td>N/A</td>
-    <td><code>posts</code></td>
-    <td>↳<code>posts.js</code></td>
-    <td>↳<code>posts.js</code></td>
-    <td>↳<code>posts.hbs</code></td>
-  </tr>
-  <tr>
-    <td><code>/posts</code></td>
-    <td><code>posts.index</code></code></td>
-    <td>↳<code>posts.js</code><br>↳<code>posts/index.js</code></td>
-    <td>↳<code>posts.js</code><br>↳<code>posts/index.js</code></td>
-    <td>↳<code>posts.hbs</code><br>↳<code>posts/index.hbs</code></td>
-  </tr>
-  <tr>
-    <td><code>/posts/new</code></td>
-    <td><code>posts.new</code></td>
-    <td>↳<code>posts.js</code><br>↳<code>posts/new.js</code></td>
-    <td>↳<code>posts.js</code><br>↳<code>posts/new.js</code></td>
-    <td>↳<code>posts.hbs</code><br>↳<code>posts/new.hbs</code></td>
-  </tr>
-</table>
+This router creates a route for `/posts` and for `/posts/new`. When a user
+visits `/posts`, they'll simply see the `posts.hbs` template. When the user
```

Comment:
I agree with the structural change to talk `index` in detail later, but I'd like it if this small intro can be accurate while not getting into the weeds.